mysite/blog/templates/blog/home.html

```diff
@@ -16,22 +16,22 @@
         <label class="slide-0">
             <h1 class="text-slide">{{ mostliked1.title }}</h1>
             <h5>{{mostliked1.content|truncatechars:70}}</h5>
-            <a href="{% url 'post-detail' mostliked1.id %}" class="readmorebutton">Read More</a>
+            <a href="{% url 'post-detail' slug=mostliked1.slug %}" class="readmorebutton">Read More</a>
```

Comment:
Hey @ManishShah120 I can see here the changes from `slug` PR. Can you please remove all these unnecessary changes?